science/raxml/Portfile

```diff
@@ -76,12 +95,12 @@ if {${build_arch} eq "i386" || ${build_arch} eq "x86_64"} {
         set EXTb "-AVX"
     }
 
-    if {![catch {sysctl hw.optional.sse3} result] && ${result} == 1} {
-        if {![catch {sysctl hw.optional.avx1_0} result] && ${result} == 1} {
-            default_variants-append +avx
-        } else {
-            default_variants-append  +sse3
-        }
+    if {![catch {sysctl hw.optional.avx2_0} result] && ${result} == 1} {
+        default_variants-append +avx2
+    } elseif {![catch {sysctl hw.optional.avx1_0} result] && ${result} == 1} {
+        default_variants-append +avx
+    } elseif {![catch {sysctl hw.optional.sse3} result] && ${result} == 1} {
+        default_variants-append  +sse3
     }
 } else {
     post-patch {
```

Comment:
not your change, but it appears the patchfiles have been used for a long time ago already, but it's still present in `files/Makefiles.patch`. Please remove this file/directory as part of the update as well.

Reply:
I believe the patch file should have been left in (see pull request #7125).